sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteTest.java

```diff
@@ -297,7 +297,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
   @Test
   public void testWriteUnbounded() {
     PCollection<String> unbounded = p.apply(CountingInput.unbounded())
-        .apply(ToString.of());
```

Comment:
I can just take out this commit from the PR if the rest looks fine. The list or the JIRA thread is probably the best place.